### PR TITLE
[Pdf_Viewer] Fix page height and width

### PR DESCRIPTION
**Pull request by AlexVelezLl**
## Summary

Fix page height and width calculation. We used `page.view` array to get width and height of the page `view[2]` as width and `view[3]` as height. But investigating by the pdfjs code, the `page.view` object seems to be a viewBox array, that means that its values describe a slice value for the box that the page belongs. So `view[2]` and `view[3]` only worked for non-sliced pages as they describe the end of the width and height (with slicing starting at 0,0), but with sliced we need to consider the start of the slices which are in `view[0]` and `view[1]`.

Before:
image

After:
image

## References

Solves #5917  

## Reviewer guidance

Go to http://localhost:8000/en/learn/#/topics/c/6f7bd64fc9655bbf99122bc5df22fb9c which has cropped pages and check that all its pages are rendering with the proper width and height.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/pdf_viewer/assets/src/views/PdfPage.vue [hunk 1]
```diff
@@ -87,13 +87,13 @@
         if (!this.pageReady) {
           return null;
         }
-        return this.pdfPage.view[3];
+        return this.pdfPage.view[3] - this.pdfPage.view[1];
```
Comment (rtibbles) [inline kolibri/plugins/pdf_viewer/assets/src/views/PdfPage.vue:93]: Could you add code comments here to add your explanation from the PR description into the code for the benefit of future readers?

File: kolibri/plugins/pdf_viewer/assets/src/views/PdfPage.vue [hunk 2]
```diff
@@ -87,13 +87,13 @@
...
       },
       actualWidth() {
         if (!this.pageReady) {
           return null;
         }
-        return this.pdfPage.view[2];
+        return this.pdfPage.view[2] - this.pdfPage.view[0];
```
Comment (rtibbles) [inline kolibri/plugins/pdf_viewer/assets/src/views/PdfPage.vue:102]: And here.

File: kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue [hunk 3]
```diff
@@ -244,8 +244,8 @@
         // Is either the first page or the saved last page visited
         const firstPageToRender = parseInt(this.getSavedPosition() * this.totalPages);
         return this.getPage(firstPageToRender + 1).then(firstPage => {
-          this.firstPageHeight = firstPage.view[3];
-          this.firstPageWidth = firstPage.view[2];
+          this.firstPageHeight = firstPage.view[3] - firstPage.view[1];
```
Comment (rtibbles) [inline kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue:250]: And here!

**Review by rtibbles — CHANGES_REQUESTED** [review]
Fix confirm - code changes make sense, just want some additional comments in the code to explain the update.

**Comment by AlexVelezLl** [comment]
Comment added! :) @rtibbles

**Review by rtibbles — APPROVED** [review]

**Pull request merged**